vividus-tests/src/main/resources/story/integration/CompositeSteps.story

```diff
@@ -0,0 +1,11 @@
+Description: Integration tests for composite steps.
+
+Meta:
+    @epic vividus-bdd-engine
+    @feature composite-steps
+
+Scenario: Verify composite step with comment
+Meta:
+    @issueId 649
+When composite with comment
+Then `${}` is equal to `value`
```

Comment:
you need to update the step and validations according to the changes in the composite step implementation

Reply:
Yep. I've just run the tests